README.md

```diff
@@ -1,17 +1,42 @@
-# Marker
+<h1 align="center">Marker</h1>
+<p align="center">
+    <br/>
+    <img height=200 src="misc/logo.png" alt="Marker logo"/>
+</p>
 
-A project trying to implement a stable linting API for Rust.
+<br/>
 
-## Contributing
+[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
+![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)
+
+**A project trying to implement a stable linting API for Rust.**
+
+> **Note**
+>
+> The project is in the very early stages of development, not everything is implemented yet.
+
+## Status
 
-The internal structure of marker and its components is documented in [`docs/internal/`](./docs/internal/).
+What works:
+- A few of example lints, which can be found in the `marker_lints` directory
+- Partial AST representation and API, both of which are in rapid development
+- Prototype `rustc` driver, to run the lints (being developed along with the API)
```

Comment:
The wording here is a bit weird, I'd say either make it a bullet point list or whole sentences. What do you think about this:

```suggestion
- A basic Item and Pattern representation is available. Expressions and types are partially implement.
- Dynamic libraries can be loaded as *lint crates*. A few example lints, can be found in the `marker_lints` directory
- The API separated from the driver and translation layer. Rustc is currently the only backend development along with the API
```